style.css

```diff
@@ -0,0 +1,211 @@
+}
+#menu-list li {
+	width: 10%;
+}
+.search-input {
+	border-radius: 12px;
+	box-shadow: inset 0.5px -0.5px 0.5px rgba(200,200,200,.4), inset 0 1px 1px rgba(0,0,0,.8);
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin-left: 5px;
+	position: relative;
+}
+.search-input .fas {
+	left: 8%;
+	position: absolute;
+	top: 25%;
+}
+.search-input input {
+	background: none;
+	border: 2px solid rgb(115,115,115);
+	border-radius: 12px;
+	height: 28px;
+	padding: 2px 18px 1px 22px;
+	width: 100%;
+}
+.search-input input:focus {
+	background-color: #fff;
+	outline: none;
+	width: 150px;
+
+	-webkit-transition: all 0.5s ease;
+	-moz-transition: all 0.5s ease;
+	-o-transition: all 0.5s ease;
+	transition: all 0.5s ease;
+}
+a {
+    text-decoration: none;
+}
+#main-links {
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+#last-link {
+	background-repeat: no-repeat;
+    height: 31px;
+    width: 100%;
+}
+#last-link:hover {
+	background-image: url(images/third-2.png);
+	background-repeat: no-repeat;
+	cursor: pointer;
+	text-decoration-line: underline;
+}
+#first-link {
+	height: 95px;
+}
+#page-banner {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+	grid-template-rows: 520px 210px;
+    overflow: hidden;
+}
+#banner-image {
+	background: url(images/your_verse_hero.jpg) 50% 97.5% no-repeat;
+	border-bottom: 2px solid #fff;
+	background-color: yellow;
+	grid-column-start: 1;
+    grid-column-end: 5;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    height: 69vh;
+    min-height: 520px;
+    max-height: 1180px;
+    width: 100%;
+}
+.promo-image {
+	border-left: 2px solid #fff;
+}
+.promo-image img {
+	height: 100%;
+	width: 100%;
+}
+#image-1 {
+	background-color: purple;
+	height: 200px;
+	grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 2;
+    grid-row-end: 3;
+}
+#image-2 {
+	background-color: cyan;
+	height: 200px;
+	grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 3;
+}
+#image-3 {
+	background-color: magenta;
+	height: 200px;
+	grid-column-start: 3;
+    grid-column-end: 4;
+    grid-row-start: 2;
+    grid-row-end: 3;
+}
+#image-4 {
+	background-color: purple;
+	height: 200px;
+	grid-column-start: 4;
+    grid-column-end: 5;
+    grid-row-start: 2;
+    grid-row-end: 3;
+}
+footer {
+	font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+	font-size: .6334em;
+    line-height: 1.2;
+}
+#footer-wrapper {
+	color: #6e6e6e;
+	margin: 0 auto;
+    padding: 1em 0 2.6em;
+    position: relative;
+    width: 90%;
+}
+#footer-links {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 10px;
+}
+.site-map-links ul {
+	margin: 0px;
+}
+.site-map-links ul li {
+	display: inline-block;
+}
+.site-map-links ul li a img {
+	position: absolute;
+	top: 8px;
+    right: -16px;
+}
```

Comment:
`SUGGESTION`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

![Screenshot of how to format code using VS Code](https://res.cloudinary.com/bolaah/image/upload/v1563045186/github-microverse-project/code-reviews/code-review002.gif)